npm/test.sh

```diff
@@ -12,6 +12,13 @@ else
     exit 1;
 fi
 popd &>/dev/null
+
+echo "Creating test files and adding paths to collection for testing form data file uploads"
+echo "Sample file 1" >> dummyFile1.txt;
```

Comment:
@shreys7 
Can we not have a check that creates the test file only when it doesn't exist and otherwise skip it, this would save the unnecessary time.
```
if [ ! -e test1.txt ];
then 
    echo "Sample file 1" >> dummyFile1.txt;
```
_Just suggesting_

Reply:
Added this @sastava007